packages/prisma/seed-app-store.ts

```diff
@@ -54,7 +54,7 @@ async function seedAppData() {
               "8988bbb8-0123-4456-b89a-b1823f70c5ff": {
                 type: "rule",
                 properties: {
-                  field: "c4296635-9f12-47b1-8153-c3a854649182",
```

Comment:
Fixes the field ID. It was wrong causing logic of first Route to be always evaluated to true for any input.

This is an issue only with the seeded-form that is used in playground testing